spec/requests/ips/delete_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe "DELETE /ips/:id", type: :request do
+  let(:user) { create(:user, :confirmed) }
+  let(:location) { create(:location, organisation: user.organisation) }
+  let!(:ip) { create(:ip, location: location) }
+
+  before do
+    https!
```

Comment:
What does this do?

Reply:
It makes the requests use HTTPS. Without this the requests are HTTP, which means they redirect to HTTPS and every response is a 307 redirect.